SoCCo/algorithms/climate.py

```diff
@@ -6,6 +6,7 @@
 import scipy.stats as sp
```

Comment:
`sp` isn't very readable. People typically do `from scipy import stats` and use `stats` afterwards.